exporter/zipkin/lib/opentelemetry/exporter/zipkin/attributes.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+# Copyright The OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+module OpenTelemetry
+  module Exporter
+    module Zipkin
+      module Attributes
+        # A count of failures when exporting Spans to Zipkin
+        OTEL_ZIPKIN_EXPORTER_FAILURE = 'otel.zipkin_exporter.failure'
```

Comment:
These are also metrics rather than attributes, and don't need the `OTEL_ZIPKIN_` prefix.